portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -5627,13 +5627,45 @@ definition {
 	}
 
 	@description = "Verify that a Form autosaves after 1 minute by default"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test WillAutosaveAfterOneMinute {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-130510 WillAutosaveAfterOneMinute pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+			
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.editDescription(formDescription = "Element Set Description");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		Pause(locator1="60000");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "Form#SAVE_NOTIFICATION",
+			value1 = "Draft");
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		AssertTextEquals(
+			locator1 = "Form#NAME_FIELD",
+			value1 = "Form 1");
+		AssertTextEquals(
+			locator1 = "Form#DESCRIPTION_FIELD",
+			value1 = "Element Set Description");
```

Comment:
We also have a macro that asserts the form description, which is `Form.viewFormDescription`.
In this case, you would use something similar to `Form.viewFormDescription(formDescription = "Form Description");`